starter.js

```diff
@@ -0,0 +1,203 @@
+//----------------Part0--------------------//
+
+var name = 'Alanoud';
+function justDoit(name ){
+      console.log(name+ ' JUST DO IT!');
+   }
+
+
+//----------------part1--------------------//
+
+function bigOrSmallString(words)
+{
+   if (words.length   > 10)
+   {
+     console.log("This word is loooooong!") ;
+}
+else {
+  console.log( " The word is short ");
+}
+}
+  bigOrSmallString(" AlanouBinSalamah");
+  bigOrSmallString(" Alanoud");
+
+
+//----------------Part2--------------------//
+
+
+function oddOrEvenString (str){
+    
+    if(str.length % 2 ){
+          console.log("This string's length is odd!") ;
+    }else {
+          console.log("This string length is TOTALLY even!") ;
+    }
+}
+
+oddOrEvenString ("Nouf");
+oddOrEvenString ("Abeer");
+
+
+//----------------Part3--------------------//
+
+                                               
+  
+var numbers = [ 34 , 16 , 10 , 70 , 45];
+
+  function medianOfArray(median){
+     median.so( function ( num1 , num2){ // I have a PROBLEM here
```

Comment:
you have some spelling mistakes with some missing `}` your function should be like this 

```
   function medianOfArray(median){
     median.sort( function ( num1 , num2){  // sort
         return num1 - num2;
       });

      var med = Math.floor(median.length/2); // use the median array 
       if(median.length % 2){
        return median[med];
     } else{
          return (median[med-1] + median[med]) / 2;
      } 
}// close your function 
```